modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+[id="images-configuration-parameters-hcp_{context}"]
+= Image controller configuration parameters for ROSA with HCP
+
+The `image.config.openshift.io/cluster` resource holds cluster-wide information about how to handle images. The canonical, and only valid name is `cluster`. It can be configured in ROSA HCP through ROSA CLI commands.
+
+[NOTE]
+====
+Parameters such as `DisableScheduledImport`, `MaxImagesBulkImportedPerRepository`, `MaxScheduledImportsPerMinute`, `ScheduledImageImportMinimumIntervalSeconds`, `InternalRegistryHostname` are not configurable.
+====
+
+[cols="3a,8a",options="header"]
+|===
+|Parameters for ROSA CLI |Description
+
+|`registry-config-allowed-registries`
+|Registries for which image pull and push actions are allowed. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are blocked.
+
+|`registry-config-insecure-registries`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+The namespace for this config map is `openshift-config`. The format of the config map is to use the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
+
+|`registry-config-blocked-registries`
+|Registries for which image pull and push actions are denied. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are allowed.
+
+|`registry-config-allowed-registries-for-import`
+|Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and
+pods. For example, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry.
+
+|`registry-config-platform-allowlist`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
```

Comment:
For this one, @andreadecorte it's currently hidden. Do we plan to expose it in the doc?

Reply:
My idea was to have this parameter not here, but in a dedicated section as it is a non functional parameter. Something like
Title: Platform allowlist
It is a list of platform mandatory registry.
It is visible in the rosa describe cluster.
It can be updated from time to time, and you will receive a Service Log with the new id. You can then update it by using rosa edit cluster --platform-allowlist-id <newid>. This will ensure you allow all the needed flows for the platform.

WDYT @chenz4027 ?

Reply:
@andreadecorte I think it's valid. It's just that the deadline is next Tuesday, which is really tight. If we can prolong the deadline I think we can squeeze it in, otherwise it might be better if we modify the description instead of introducing a new section.

Reply:
ok then quick proposal, at minimum we should:
1. move this as last parameter of the table
2. description
```
A list of Red Hat registries is automatically allowed. This list can be periodically updated and impacted clusters will receive a notification with the new allowlist id. In such case, user needs to use this parameter to update to the expected id to the expected one 
```

Reply:
moved and new description added

Reply:
moved the parameter to end and new description provided by @andreadecorte thanks